src/aatoolbox/datasources/datasource.py

```diff
@@ -60,16 +60,23 @@ def __init__(
             is_public=is_public, is_raw=False, is_global=is_global_processed
         )
 
+    @staticmethod
     def _config_attribute_name_validator(
-        self, config_datasource_name: str, country_config: CountryConfig
+        config_datasource_name: str, country_config: CountryConfig
     ):
         try:
-            return getattr(country_config, config_datasource_name)
-        except AttributeError as err:
+            datasource_config = getattr(country_config, config_datasource_name)
+        except AttributeError:
+            datasource_config = None
+        # If the datasource is one of the defaults, it's set to None and
+        # thus an attribute error won't be raised. So also need to check for
+        # the case when it is None.
+        if datasource_config is None:
             raise AttributeError(
                 f"{config_datasource_name} needs to be added to the the "
```

Comment:
very minor, but double "the" in error string

Reply:
Oh man I do this this all the time, good catch